syntax = "proto3";
import "FlatInstancedItem.proto";
import "DisplayName.proto";

message ProfileData {
	string playerId = 1;
	DisplayName displayName = 2;
	FlatInstancedItem banner = 3;
	string crewScore = 4;
	double currentSoloRank = 5;
	double highestTeamRank = 6;
	string divisionType = 7;
}